README.md

```diff
@@ -36,6 +36,19 @@ By default autoscaling configuration is automatically applied to all preconfigur
 Still, we can exclude individual tables or tweak their configuration via configuration within`serverless.yml` config:
 
 ```yaml
+resources:
+  Resources:
+    Table1:
+      Properties:
+        TableName: tableName1
+    Table2:
+      Properties:
+        TableName: tableName2
+    ...
+    Table7:
+      Properties:
+        TableName: 
+          Fn::Sub: ${AWS::Region}-table
```

Comment:
That's indeed a case that was not taken into account. Still wouldn't (in this specific case) following work out of a box (?)

```yaml
 TableName: ${AWS::Region}-table
```

With important note that `serverless-pseudo-parameters` needs to be loaded before this plugin

Reply:
We're using `serverless-cf-vars` which does something similar, but the gotcha here is that `TableName: #{AWS::Region}-table` gets turned into:
```
TableName:
  Fn::Sub: ${AWS::Region}-table
```
Which is an object and not handled gracefully by the minimatch pattern matching